capstone-frontend/src/components/BookSearchTile.jsx

```diff
@@ -1,20 +1,7 @@
 import React from 'react';
-import { Container, Row, Col, DropdownButton, Dropdown } from 'react-bootstrap';
-import '../styles/BookSearchTile.css';
-
```

Comment:
The changes in ```BookSearchTile.jsx``` will most probably result in a merge conflict. We should discuss it before merging.

Reply:
Discussed and resolved all conflicts!